### PR TITLE
UIDEXP-88: Integrate new mapping profile form with backend

### PR DESCRIPTION
**Pull request by YevheniiMaltsev**
## Purposes

Integrate new mapping profile form with backend side. [Story](https://issues.folio.org/browse/UIDEXP-88)

## Screenshots

Screen Shot 2020-05-28 at 11 18 16
Screen Shot 2020-05-28 at 11 18 22

### CODE REVIEW

File: src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js [hunk 1]
```diff
@@ -81,10 +78,10 @@ const MappingProfilesContainer = ({
 
 MappingProfilesContainer.propTypes = {
   match: matchShape.isRequired,
-  mutator: PropTypes.object.isRequired,
   history: historyShape.isRequired,
   location: locationShape.isRequired,
   resources: PropTypes.object.isRequired,
+  mutator: PropTypes.shape({ mappingProfiles: PropTypes.shape({ POST: PropTypes.func }) }).isRequired,
```
Comment (VictorSoroka1) [inline src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js:84]:
```suggestion
  mutator: PropTypes.shape({ mappingProfiles: PropTypes.shape({ POST: PropTypes.func.isRequired }) }).isRequired,
```

File: package.json [hunk 2]
```diff
@@ -56,7 +56,8 @@
     "react-intl": "^4.5.1",
     "react": "*",
     "react-router": "*",
-    "react-router-dom": "*"
+    "react-router-dom": "*",
+    "@folio/react-intl-safe-html": "^2.0.0"
```
Comment (VictorSoroka1) [inline package.json:60]: Please place it near stripes so we have folio deps grouped.

File: src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js [hunk 3]
```diff
@@ -97,7 +94,7 @@ MappingProfilesContainer.manifest = Object.freeze({
     records: 'mappingProfiles',
     recordsRequired: '%{resultCount}',
     perRequest: RESULT_COUNT_INCREMENT,
-    clientGeneratePk: false,
+    clientGeneratePk: true,
```
[the remark this answers is not in the document]
Comment (YevheniiMaltsev) [reply src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js:97]: This is how the module works. The same behavior is applied for other modules. And if we turn off this flag, the backend will throw an error.

[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js:97]: Good for now, I think we should ask our backend folks to generate id on their side.

[the remark this answers is not in the document]
Comment (YevheniiMaltsev) [reply src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js:97]: Yes, I was surprised to see this behavior

**Pull request merged**